db/migrate/20220103224548_create_transactions.rb

```diff
@@ -1,8 +1,8 @@
 class CreateTransactions < ActiveRecord::Migration[5.2]
   def change
     create_table :transactions do |t|
-      t.references :invoices, foreign_key: true
-      t.integer :credit_card_number
+      t.references :invoice, foreign_key: true
+      t.string :credit_card_number
```

Comment:
Good call on changing this to a String. As an Int we can't fit in all 16 characters. As well we shouldn't have to do any math on the CC number either.